logr.go

```diff
@@ -43,7 +43,10 @@ limitations under the License.
 //
 // Info() and Error() are very similar, but they are separate methods so that
 // LogSink implementations can choose to do things like attach additional
-// information (such as stack traces) on calls to Error().
+// information (such as stack traces) on calls to Error(). Another difference
+// is that error messages always get emitted, regardless of the current
+// verbosity. This can also be used when no error instance is available
+// by calling Error with nil as err parameter.
```

Comment:
I'd cut words:

"""
...on calls to Error().  Error() messages are always logged, regardless of the current verbosity.  If there is no error instance available, passing nil is valid.
"""